bedrock/newsletter/templates/newsletter/existing.html

```diff
@@ -99,11 +95,18 @@ <h1>{{ _('Manage your Email Preferences') }}</h1>
             {{ form['format'] }}
           </div>
           {{ form['format'].errors }}
+          <p class="disclaimer">{{ _('Text subscribers will receive an email twice a year to confirm continuation of the subscription. Those emails may include HTML.') }}</p>
+        </div>
+        <div class="field">
+          <p>{{ _('Many of our communications are related to an account you’ve signed up for, such as Firefox Accounts, Mozilla Developer Network (MDN), or AMO. To manage one of your accounts or see a list of all the accounts visit our <a href="%s">account management support page</a>.')|format('https://support.mozilla.org/kb/managing-account-data') }}
```

Comment:
Should "AMO" here be expanded (like MDN)?

Reply:
Good question. I'll ask in the copy doc.

Reply:
Updated to `Add-on Developer`.